api/tests/integration/domain/services/organization-learners-xml-service_test.js

```diff
@@ -169,5 +169,21 @@ describe('Integration | Services | organization-learnerz-xml-service', function
       //then
       expect(error).to.be.instanceof(SiecleXmlImportError);
     });
+    context('when student is born in France', function () {
```

Comment:
Il manque le cas 
when student is born in France => should not reject with missing birth city code

Reply:
Les cas passants sont déjà testés dans le premier test

Reply:
Mais la c'est pas forcement un cas passant

Reply:
Ah ok donc tu veux plutôt dire when student is NOT born in France => should not reject with missing birth city code, je n'avais pas compris :sweat_smile:

Reply:
ce cas est aussi couvert par le premier test. On lui passe deux étudiants valides, un français avec un code de ville et un étranger sans code de ville